package.json

```diff
@@ -11,7 +11,7 @@
     "test-ci": "npm test",
     "test:watch": "npm test -- --watchAll",
     "test": "jest \"\\.spec\\.ts\"",
-    "test-integrations": "jest \"\\.integration\\.ts\" --forceExit",
+    "test-integrations": "jest \"\\.integration\\.ts\" --forceExit --runInBand",
```

Comment:
`--runInBand` runs the integration test suites in sequential order.

otherwise, the integrations test runs in parallel and userEventHandler tests pulls prospect events with the function `getGoodSnowplowEvents`